.vitepress/config.ts

```diff
@@ -86,39 +90,43 @@ function getGuideSidebar() {
         {
          text: 'Queries',
          link: '/guide/advanced/queries',
-        },
+        }
```

Comment:
```suggestion
        },
```

A trailing comma is considered a good practice in TypeScript and in Rust.